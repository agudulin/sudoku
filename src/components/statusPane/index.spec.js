/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'

import StatusPane from './'

describe('StatusPane', () => {
  it('renders correctly on error', () => {
    const tree = renderer.create(
      <StatusPane />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('renders correctly on game over', () => {
    const tree = renderer.create(
      <StatusPane gameOver validBoard />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
