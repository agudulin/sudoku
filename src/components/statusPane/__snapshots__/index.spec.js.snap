exports[`StatusPane renders correctly on error 1`] = `
<div
  className="status-pane status-pane--active">
  ğŸ‘º Error! There is a conflict.
</div>
`;

exports[`StatusPane renders correctly on game over 1`] = `
<div
  className="status-pane status-pane--active">
  ğŸ‘¾ Congratulations!
</div>
`;
